src/pages/Editor.vue

```diff
@@ -445,10 +443,26 @@ export default {
       var classObject = [
         { 'cursor-not-allowed': this.isPublished },
 
-        `rounded-md font-bold p-5 h-12 bg-primary-button
+        `rounded-md font-bold p-5 h-12 w-96 bg-primary-button
```

Comment:
increased the button size
this looks very easily accessible in mobile - try the 360px width option